mods/ts/sequences/infantry.yaml

```diff
@@ -45,110 +47,263 @@ e1.gdi:
+	prone-attack:
 		Start: 212
 		Length: 6
 		Facings: 8
 		ShadowStart: 504
-	standup-0:
-		Start: 260
-		Length: 2
-		Facings: 8
-		ShadowStart: 552
-	die6: electro
-		Length: *
+
+e1.gdi:
+	Inherits: ^e1
+	Inherits@attack: ^e1attack
+	Defaults: e1
 	icon: sidebar-gdi|e1icon
 
 e1.nod:
+	Inherits: ^e1
+	Inherits@attack: ^e1attack
 	Defaults: e1
+	icon: sidebar-nod|e1icon
+
+e2:
+	Inherits: ^e1
+	Inherits@attack: ^e1attack
+	icon: e2icon
+
+e3:
+	Inherits: ^e1
+	Inherits@attack: ^e1attack
+	icon: e4icon
+
+engineer.gdi:
+	Inherits: ^e1
+	Defaults: engineer
+	icon: sidebar-gdi|engnicon
+
+engineer.nod:
+	Inherits: ^e1
+	Defaults: engineer
+	icon: sidebar-nod|engnicon
+
+umagon:
+	Inherits: ^e1
+	Inherits@attack: ^e1attack
+	icon: umagicon
+
+ghost:
+	Inherits: ^e1
+	Inherits@attack: ^e1attack
+	icon: gosticon
+
+mhijack:
+	Inherits: ^e1
+	icon: chamicon
+
+chamspy:
+	Inherits: ^e1
+	icon: chamicon
+
+mutant:
+	Inherits: ^e1
+	Inherits@attack: ^e1attack
+	icon: mutcicon
+
+mwmn:
+	Inherits: ^e1
+	Inherits@attack: ^e1attack
+	icon: mutcicon
+
+mutant3:
+	Inherits: ^e1
+	Inherits@attack: ^e1attack
+	icon: mutcicon
+
+tratos:
+	Inherits: ^e1
+	icon: mutcicon
+
+oxanna:
+	Inherits: ^e1
+	Inherits@attack: ^e1attack
+
+slav:
+	Inherits: ^e1
+	Inherits@attack: ^e1attack
+
+civ1:
+	Inherits: ^e1
+	attack:
+		Start: 164
+		Length: 6
+		Facings: 8
+		ShadowStart: 456
+	panic-attack:
+		Start: 164
+		Length: 6
+		Facings: 8
+		ShadowStart: 456
+	panic-run:
+		Start: 86
+		Length: 6
+		Facings: 8
+		ShadowStart: 378
+	panic-stand:
+		Start: 86
+		Facings: 8
+		Stride: 6
+		ShadowStart: 378
+
+civ2:
+	Inherits: ^e1
+	panic-run:
+		Start: 86
+		Length: 6
+		Facings: 8
+		ShadowStart: 378
+	panic-stand:
+		Start: 86
+		Facings: 8
+		Stride: 6
+		ShadowStart: 378
+
+civ3:
+	Inherits: civ1
+
+cyborg:
+	Defaults:
 		Tick: 80
 	stand:
 		Facings: 8
-		ShadowStart: 292
+		ShadowStart: 370
 	run:
 		Start: 8
 		Length: 6
 		Facings: 8
-		ShadowStart: 300
+		ShadowStart: 378
 	idle1:
 		Start: 56
 		Length: 15
-		ShadowStart: 348
+		ShadowStart: 426
 	idle2:
 		Start: 71
 		Length: 15
-		ShadowStart: 363
-	prone-run:
+		ShadowStart: 441
+	crippled-run:
 		Start: 86
 		Length: 6
 		Facings: 8
-		ShadowStart: 378
-	prone-stand:
+		ShadowStart: 456
+	crippled-stand:
 		Start: 86
 		Facings: 8
 		Stride: 6
-		ShadowStart: 378
+		ShadowStart: 456
+	attack:
+		Start: 164
+		Length: 6
+		Facings: 8
+		ShadowStart: 534
+	crippled-attack:
+		Start: 212
+		Length: 6
+		Facings: 8
+		ShadowStart: 582
+	emp-overlay: emp_fx01
+		Length: *
+		ZOffset: 512
+		BlendMode: Additive
+	icon: cybiicon
+
+cyc2:
+	stand:
+		Facings: 8
+		ShadowStart: 308
+	run:
+		Start: 8
+		Length: 9
+		Facings: 8
+		ShadowStart: 316
+	idle1:
+		Start: 80
+		Length: 15
+		ShadowStart: 388
+	idle2:
+		Start: 95
+		Length: 15
+		ShadowStart: 403
+	crippled-run:
+		Start: 110
+		Length: 9
+		Facings: 8
+		ShadowStart: 418
+	crippled-stand:
+		Start: 110
+		Facings: 8
+		Stride: 9
+		ShadowStart: 418
 	die1:
```

Comment:
What's the point in keeping/editing these? You removed WithDeathAnimation entirely from cyborgs with the new template.